test/oas3/execute/style-explode/query.js

```diff
@@ -781,7 +781,7 @@ describe('OAS 3.0 - buildRequest w/ `style` & `explode` - query parameters', ()
 
         expect(req).toEqual({
           method: 'GET',
-          url: `/users?id=3 4 5 ${SAFE_INPUT_RESULT}`,
+          url: `/users?id=3%204%205%20${SAFE_INPUT_RESULT}`,
```

Comment:
According to the example table, when style is `spaceDelimited`, the value is delimited by `%20`.

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#style-examples